wiki/Tournaments/The_Roundtable/1v1_double_elimination/en.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - TRT
+  - The Roundtable
+  - The Round table
+---
+
+# The Roundtable 1v1 double elimination
+
+![The Roundtable banner](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+[**The Roundtable**](/wiki/Tournaments/The_Roundtable)'s **1v1 double elimination tournament** takes place in the second half of the second day and the entirety of the third day of the event.
+
+## Prizes
```

Comment:
Is there a specific reason why this is inverted?

Reply:
not sure what "inverted" means here, could you explain?

Reply:
oh i see what they meant now. this one goes in order 3rd-2nd-1st when usually it's the opposite